Conversion-Class Temperatura

```diff
@@ -0,0 +1,16 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void conversionfromFtoC () {
+        double setTemperatura = 5/9*(this.temperatura-32);
+    }
```

Comment:
This is my main now 

package com.company;

public class Main {

    public static void main(String[] args) {
        // write your code here
//
//        Elev johnny = new Elev();
//        johnny.setNume("Ionut");
//
//        Elev maria = new Elev();
//        maria.setNume("Maria");
//
//        System.out.println("Pe primul elev il cheama:" +
//                johnny.getNume());
//        System.out.println("Pe al doilea elev il cheama:" +
//                maria.getNume());
//
//        johnny.setGender('B');
//        maria.setGender('F');
//
//        System.out.println("Primul elev este:" +
//                johnny.getGender());
//        System.out.println("Al doilea elev este:" +
//                maria.getGender());
//
//        johnny.setVarsta(15);
//        maria.setVarsta(12);
//
//        System.out.println("Primul elev are varsta de:" +
//                johnny.getVarsta());
//        System.out.println("Al doilea elev are varsta de:" +
//                maria.getVarsta());
//
//        johnny.getNotaMate(2);
//        maria.getNotaMate(5);
//
//        System.out.println("Elevul " + johnny.getNume() + " are nota: " +
//                johnny.getNotaMate());
//        System.out.println("Elevul " + maria.getNume() + " are nota: " +
//                maria.getNotaMate());
//
//
        Temperatura conversionfromFtoC = new Temperatura ();
        conversionfromFtoC.setTemperatura(50);
        System.out.println("Temperatura zilei de azi este de: " + conversionfromFtoC + " grade celsius.");
    }
}